pkg/policy/k8s/watcher.go

```diff
@@ -55,124 +49,6 @@ type policyWatcher struct {
 	// cidrGroupPolicies is the set of policies that contain ToServices references
 	toServicesPolicies map[resource.Key]struct{}
 	cnpByServiceID     map[k8s.ServiceID]map[resource.Key]struct{}
-	// maps cnp file name to cnp object. this is required to retrieve data during delete.
-	fileNameToCnpCache map[string]*types.SlimCNP
-}
-
-func (p *policyWatcher) translateToCNPObject(file string) (*cilium_v2.CiliumNetworkPolicy, error) {
```

Comment:
If possible, please squash your commits into one, it makes it easier to review what changes are actual changes and which ones are just undoing prior approaches.

Reply:
the reason i didn't squash, thought it would be easier to review changes by commit.. if not, I would squash it